.release/ci.hcl

```diff
@@ -175,6 +175,18 @@ event "verify" {
   }
 }
 
+event "enos-verify-stable" {
+    depends = ["verify"]
+      action "enos-verify-stable" {
```

Comment:
The formatting is strange for this event

Reply:
Yeah good catch.  My editor had 4space setting then.